.circleci/config.yml

```
@@ -0,0 +1,43 @@
+version: 2.1
+
+aliases:
+  - &pnpm-cache-key pnpm-packages-{{ checksum "client/pnpm-lock.yaml" }}
```

```suggestion
  - &pnpm-cache-key pnpm-packages-v1-{{ checksum "client/pnpm-lock.yaml" }}
```
同じhashでもキャッシュクリアしたい時があるかもなので、バージョンを挟んでくれると助かるかも

---

`v1` の数字をインクリメントするときはキャッシュを強制的にクリアしたいとき、であれば、それはあるpnpm-lock.yamlにおけるキャッシュの試行回数であると考えられます。

それが pnpm-lock.yaml のバージョンである checksum の値よりも上に表記されるのは変だと思います。

`name-{{checksum}}-1` の方が良いのではないでしょうか？

---

https://circleci.com/docs/ja/caching/#managing-caches
CircleCIの慣習として、cache keyの先頭にバージョン名をつけるとキャッシュ全体をクリアできるので、先頭に`v1`のようにprefixをつける形でお願いしたいです。